src/core/js/adapt.js

```diff
@@ -106,9 +106,11 @@ define([
         }
 
         var navigationHeight = $(".navigation").outerHeight();
+        // prevent scroll issue when component description aria-label coincident with top of component
+        var ariaLabelBuffer = $(selector.is('.component')) ? 5 : 0;
```

Comment:
I know it's not ideal and we have other magic numbers with some of the deferred calls, particularly in the a11y code. There is an alternative, which I discussed with @kirsty-hames and that is to place this 5px offset in the theme to shove the aria-label down. I preferred the code method as it means we don't need to make this change in multiple themes.

@brian-learningpool, @kirsty-hames, @tomgreenfield is the consensus that this is better fixed in the theme(s)?